__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const app = require('../app');
+const seed = require("../db/seeds/seed");
+const request = require('supertest');
+const db = require('../db/connection');
+const data = require("../db/data/test-data");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET/api/topics", () => {
+    it("returns a status code of 200", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200);
+    });
+    it("should return an object with a key of topics", () => {
+        return request(app)
+        .get("/api/topics")
+        .then(({body}) => {
+            expect(body).toHaveProperty("topics");
+        });
+    });
```

Comment:
I like the granularity of the first test checking the status code! Though I think this second one is slight overkill. All of your subsequent tests are going to be using the `topics` property and thus checking its existence by default